libs/designer/src/lib/ui/mcp/parameters/EditOperation.tsx

```diff
@@ -348,31 +371,12 @@ export const EditOperation = () => {
 
   return (
     <div className={styles.container}>
-      {/* Operation Header */}
-      <div className={styles.operationHeader}>
-        <div className={styles.operationIcon}>
-          <SettingsIcon />
-        </div>
-        <div className={styles.operationInfo}>
-          <Text size={500} className={styles.operationTitle}>
-            {metadata?.summary ?? selectedOperationId}
-          </Text>
-          <Text className={styles.operationMeta}>
-            {operationInfo.type} • {operationInfo.connectorId}
-          </Text>
-        </div>
-        <div className={styles.statusBadge}>
-          <Badge appearance="filled" icon={<CheckmarkCircle16Regular />} className={styles.savedBadge}>
-            {INTL_TEXT.saved}
-          </Badge>
-        </div>
-      </div>
-
-      <Divider className={styles.divider} />
-
       {/* Description Section */}
       <div className={styles.section}>
-        <Field label={INTL_TEXT.descriptionLabel} className={styles.descriptionField}>
+        <Text size={400} weight="semibold" className={styles.descriptionField}>
```

Comment:
nit: can we make it so that the description takes the whole width of the panel to align with the parameters